### ISSUE TITLE
When terraform expects the variables, it's stuck

### ISSUE THREAD

**Issue by ianomad** [coinbase/terraform-landscape#103]
cmd: terraform plan .. | landscape...
cmd: <here it expects variable names>, but it's stuck

i guess the workaround is using -var flags, but that's a bit inconvenient

**Comment by sds** [comment]
Would happily accept a pull request adding support for this!

**Comment by xild** [comment]
@sds any suggestion to implement? I can go over it if you have an idea how to do that in a good way 😄

**Comment by ianomad** [comment]
i am gonna guess the reason why it's stuck is because when doing a pipe, landscape input takes over the stuck terraform process waiting for its input. one way is let landscape read those vars (not sure how challenging that will be)

**Comment by Dzhuneyt** [comment]
This is a showstopper.

**Comment by sds** [comment]
Yes, @ianomad is correct about the situation with standard input and piping from `terraform` to `landscape`. 

I'm going to lock this thread to prevent comments that add noise. 

At this point, would welcome a pull request adding support for this!
